test/jmap/mail/calendar/reply/calendar_event_reply_method_test.dart

```diff
@@ -0,0 +1,119 @@
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+import 'package:jmap_dart_client/jmap/jmap_request.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/reply/calendar_event_reply_method.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/reply/calendar_event_reply_response.dart';
+
+void main() {
+  final baseOption = BaseOptions(method: 'POST');
+  final dio = Dio(baseOption)..options.baseUrl = 'http://domain.com/jmap';
+  final dioAdapter = DioAdapter(dio: dio);
+  final dioAdapterHeaders = {"accept": "application/json;jmapVersion=rfc-8621"};
+  final httpClient = HttpClient(dio);
+  final processingInvocation = ProcessingInvocation();
+  final requestBuilder = JmapRequestBuilder(httpClient, processingInvocation);
+  final accountId = AccountId(Id('123abc'));
+  final blobIds = [Id('abc123')];
+  final methodCallId = MethodCallId('c0');
+
+  Map<String, dynamic> constructData(CalendarEventReplyMethod method) => {
+    "using": method.requiredCapabilities
+      .map((capability) => capability.value.toString())
+      .toList(),
+    "methodCalls": [
+      [
+        method.methodName.value,
+        {
+          "accountId": accountId.id.value,
+          "blobIds": [blobIds.first.value],
+        },
+        methodCallId.value
+      ]
+    ]
+  };
+
+  Map<String, dynamic> constructSuccessResponse(CalendarEventReplyMethod method) => {
+    "sessionState": "abcdefghij",
+    "methodResponses": [[
+      method.methodName.value,
+      {
+        "accountId": accountId.id.value,
+        "accepted": [blobIds.first.value],
+        "rejected": [blobIds.first.value],
+        "maybe": [blobIds.first.value],
+      },
+      methodCallId.value
+    ]]
+  };
+
+  Map<String, dynamic> constructFailureResponse(CalendarEventReplyMethod method) => {
+    "sessionState": "abcdefghij",
+    "methodResponses": [[
+      method.methodName.value,
+      {
+        "accountId": accountId.id.value,
+        "notCreated": [blobIds.first.value],
+        "notRejected": [blobIds.first.value],
+        "notMaybe": [blobIds.first.value],
+      },
+      methodCallId.value
+    ]]
+  };
+
+  Future<EventActionResponse?> arrangeAndExecuteTest(
+    CalendarEventReplyMethod method,
+    EventActionResponse Function(Map<String, dynamic> json) deserialize,
+    {bool failure = false}
+  ) async {
+    // arrange
+    final invocation = requestBuilder.invocation(method, methodCallId: methodCallId);
+    dioAdapter.onPost(
+      '',
+      (server) => server.reply(200, failure
+        ? constructFailureResponse(method)
+        : constructSuccessResponse(method)),
+      data: constructData(method),
+      headers: dioAdapterHeaders,
+    );
+
+    // act
+    return (await (requestBuilder..usings(method.requiredCapabilities))
+      .build()
+      .execute())
+        .parse<EventActionResponse>(invocation.methodCallId, deserialize);
+  }
+
+  group('calendar event accept method', () {
```

Comment:
Write unit tests for cases
- [ ] has both success and failure events.

Reply:
Done